Packages/com.unity.inputsystem/Documentation~/Actions.md

```diff
@@ -88,7 +77,7 @@ public class ExampleScript : MonoBehaviour
 
 The result is similar to using an Actions defined in the Input Actions editor, except the Actions are defined in the GameObject's properties and saved as Scene or Prefab data, instead of in a dedicated Asset.
 
-When you embed actions in a MonoBehaviour and assign that MonoBehaviour to a GameObject, the GameObject's Inspector window displays an interface similar to the Actions Asset window, which allows you to set up the bindings for those actions. For example:
+When you embed actions in a MonoBehaviour and assign that MonoBehaviour to a GameObject, the GameObject's Inspector window displays an interface similar to the Action Asset window, which allows you to set up the bindings for those actions. For example:
```

Comment:
This "embed actions" is fine I guess but very vague what it means. I would rather suggest, 
"When you embed actions by defining serialised InputAction fields in a MonoBehavior....".
Might point out that for this scenario, the serialised action configuration is stored with the parent object as part of the scene, opposite to being serialised with an InputActionAsset.

Reply:
fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)